angular-hometask/src/app/core/core.module.ts

```diff
@@ -5,19 +5,25 @@ import { FooterComponent } from './footer/footer.component';
 import { BreadcrumbsComponent } from './breadcrumbs/breadcrumbs.component';
 import { LogoComponent } from './logo/logo.component';
 import { PickABooComponent } from './pick-a-boo/pick-a-boo.component';
-
-
+import { PageNotFoundComponent } from './page-not-found/page-not-found.component';
 
 @NgModule({
-  declarations: [HeaderComponent, FooterComponent, BreadcrumbsComponent, LogoComponent, PickABooComponent],
+  declarations: [
+    HeaderComponent,
```

Comment:
U can create array and assign to declarations and exports

Reply:
Do you mean this? fc870c63cb9ba30558849679d9ff6a79e39d4e36